src/components/UI/Tab/Tab.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import styles from './Tab.module.scss';
+
+export const Tab = ({label, onClick}) => {
+  return (
+    <button className={styles.tab} onClick={() => onClick()}>
```

Comment:
No need for the anonymous function.
`onClick={onClick}`

Reply:
done